README.md

```diff
@@ -137,29 +68,46 @@ try {
 }
 ```
 
-### OpenCensus extension
+## Customization
 
-The `opencensus` extension collects nested span data throughout the course of your application's
-execution. You can collect spans in 2 ways, by watching for function/method invocations or by manually
-starting and stopping spans. In both cases, the spans will be collected together and can be retrieved
-at the end of the request.
+### Samplers
 
-See [extension README](ext/README.md) for more information.
+You can specify different samplers when initializing a tracer. The default
+sampler is the `AlwaysSampleSampler` which will attempt to trace all requests.
 
-## Versioning
+The provided samplers are:
+
+| Class | Description |
+| ----- | ----------- |
+| [NeverSampleSampler][never-sampler] | Never trace any requests |
+| [AlwaysSampleSampler][always-sampler] | Trace all requests |
+| [QpsSampler][qps-sampler] | Trace X requests per second. Requires a PSR-6 cache implementation |
+| [ProbabilitySampler][probability-sampler] | Trace X percent of requests. |
 
-You can retrieve the version of this extension at runtime.
+If you would like to provide your own sampler, create a class that implements
+`SamplerInterface`.
 
-```php
-/**
- * Return the current version of the opencensus_trace extension
- *
- * @return string
- */
-function opencensus_trace_version();
-```
+### Exporters
+
+You can choose different exporters to send the collected traces to.
+
+The provided exporters are:
+
+| Class | Description |
+| ----- | ----------- |
+| [EchoExporter][echo-exporter] | Output the collected spans to stdout |
+| [FileExporter][file-exporter] | Output JSON encoded spans to a file |
+| [StackdriverExporter][stackdriver-exporter] | Report traces to Google Cloud Stackdriver Trace |
+| [LoggerExporter][logger-exporter] | Exporter JSON encoded spans to a PSR-3 logger |
+| [NullExporter][null-exporter] | No-op |
+| [ZipkinExporter][zipkin-exporter] | Report collected spans to a Zipkin server |
+
+If you would like to provide your own reporter, create a class that implements
+`ExporterInterface`.
 
-This library follows [Semantic Versioning](http://semver.org/).
+## Versioning
+
+This library follows [Semantic Versioning][semver].
 
 Please note it is currently under active development. Any release versioned
 0.x.y is subject to backwards incompatible changes at any time.
```

Comment:
Nit: It is not clear from this doc that which state this library is. I assume it's alpha, but there's no mention

Reply:
Added a badge image that shows the latest released packagist version and added a Status: Pre-Alpha below the descriptions of the general status.